packages/test-worker/src/test-worker-local.ts

```diff
@@ -0,0 +1,104 @@
+import * as process from 'process';
+import * as path from 'path';
+import { EventEmitter } from 'events';
+import { Sandbox } from '@testring/sandbox';
+import { testAPIController } from '@testring/api';
+import {
+    ITransport,
+    ITestExecutionMessage,
+    ITestExecutionCompleteMessage,
+    TestWorkerAction,
+    TestStatus,
+    TestEvents,
+    ITransportChild,
+    ITransportDirectMessage,
+} from '@testring/types';
+
+export class TestWorkerLocal extends EventEmitter implements ITransportChild {
```

Comment:
I think it's really weird name for interface, maybe just `IWorker` or something like this?